lib/tasks/reminder.rake

```diff
@@ -0,0 +1,21 @@
+namespace :reminder do
+  desc "Send actionable approver reminder email for pending requests"
```

Comment:
I am not sure why there is a need for this because there is no linked tracker story or test. Can we get one or both of those?